examples/demo-rollup/benches/prover/datagen.rs

```diff
@@ -1,25 +1,29 @@
+use sov_data_generators::bank_data::BankMessageGenerator;
+use sov_data_generators::MessageGenerator;
+use sov_demo_rollup::MockDemoRollup;
 use sov_mock_da::{MockAddress, MockBlock, MockDaService};
-use sov_rng_da_service::{generate_create_token_payload, generate_transfers};
 use sov_rollup_interface::services::da::DaService;
 
+const DEFAULT_TRANSFER_NUM: u64 = 100;
```

Comment:
can we have an env var override so CI can set 1 transfer, 1 block? might be good to make it configurable

Reply:
Solved in the last commit. I added env variable support and added it to the CI configuration with a fallback if the env variables are not defined.